### PR TITLE
Remove experimental /call functionality from Connect ✂️✂️✂️

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Refactor

## Description

This PR removes the experimental `/call` functionality and all related code from Connect. To quote the issue:

> [T]his is an area of the codebase where the road to supporting this not-fully-baked feature is unclear, and removal will result in some wins.
> 
> Product team has synced and agreed that this is okay. 

## Related Tickets & Documents

https://github.com/forem/internalEngineering/issues/399

## QA Instructions, Screenshots, Recordings

All specs should pass, not much else to QA.

### UI accessibility concerns?

n/a

## Added tests?

- [X] No, and this is why: only removed code

## [Forem core team only] How will this change be communicated?

- [X] I will share this change internally with the appropriate teams

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -95,7 +95,8 @@
       get "/badges/badge_achievements", to: redirect("/admin/badge_achievements")
       get "/badges/badge_achievements/award_badges", to: redirect("/admin/badge_achievements/award_badges")
 
-      # NOTE: @ridhwana These routes below will be deleted once we remove the admin_restructure feature flag, hence they've been regrouped them in this manner.
+      # NOTE: @ridhwana These routes below will be deleted once we remove the
```
Comment (citizen428) [inline config/routes.rb:98]: This change is completely unrelated to this PR, but I had a Rubocop error when merging back `master` to resolve a conflict, so I fixed it.

**Review by rhymes — COMMENTED** [review]
There are still a few pieces to remove:

https://github.com/forem/forem/blob/18a0fd5f61dd545655c72e2a4ecb7dbedcdbb001/package.json#L180

and this file IIRC

https://github.com/forem/forem/blob/master/app/javascript/packs/videoChat.jsx

**Comment by citizen428** [comment]
Thanks, @rhymes!

image

**Review by rhymes — APPROVED** [review]
LGTM!

**Comment by rhymes** [comment]
Added scissors in the title so it gets merged faster :D

**Review by jacobherrington — APPROVED** [review]
✂️

**Pull request merged**